corporate/lib/stripe.py

```diff
@@ -556,7 +556,7 @@ def is_sponsored_or_pending(self, customer: Optional[Customer]) -> bool:
         pass
 
     @abstractmethod
-    def update_context_initial_upgrade(self, context: Dict[str, Any]) -> None:
+    def get_specific_context_upgrade_page(self, context: Dict[str, Any]) -> None:
```

Comment:
`add_extra_context_for_upgrade_page` might be clearer, since this extends the `context` dictionary?

Reply:
Changed to a name aligned with https://github.com/zulip/zulip/pull/27868#discussion_r1404468929, after noting that the way it's used now, `get_` is appropriate.